src/alpine/3.18/helix/Dockerfile

```diff
@@ -52,5 +53,5 @@ USER helixbot
 # Install Helix Dependencies
 ENV VIRTUAL_ENV=/home/helixbot/.vsts-env
 RUN python3 -m venv $VIRTUAL_ENV
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+ENV PATH="${VIRTUAL_ENV}/bin:$PATH"
```

Comment:
I notice inconsistent usage of this syntax. In the AlmaLinux Dockerfile, it used `${PATH}` but here it does not. I think they should all be consistent.

Reply:
sure. expect in the next iteration (likely tomorrow at best)